scripts/bamboo/build_and_test.sh

```diff
@@ -17,23 +17,21 @@ function trycmd
   fi
 }
 
-SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
-
-export UMPIRE_DIR=$(git rev-parse --show-toplevel)
 export BUILD_DIR=build-${SYS_TYPE}
 
+rm -rf ${BUILD_DIR} 2> /dev/null
+mkdir -p ${BUILD_DIR} 2> /dev/null
+cd ${BUILD_DIR}
+
 export COMPILER=${1:-gcc_4_9_3}
 export BUILD_TYPE=${2:-Release}
 
-mkdir ${BUILD_DIR} 2> /dev/null
-cd ${BUILD_DIR}
-
 echo "Configuring..."
 
 trycmd "cmake -DENABLE_DEVELOPER_DEFAULTS=On \
-	  -C ${UMPIRE_DIR}/.gitlab/conf/host-configs/${SYS_TYPE}/${COMPILER}.cmake \
-	  -C ${UMPIRE_DIR}/host-configs/${SYS_TYPE}/${COMPILER}.cmake \
-	  -DCMAKE_BUILD_TYPE=${BUILD_TYPE} ${BUILD_OPTIONS} ../"
+	  -C ../src/.gitlab/conf/host-configs/${SYS_TYPE}/${COMPILER}.cmake \
```

Comment:
I could see this being brittle when creating new plans. Can the source directory be identified some other way?

Reply:
Good idea